### ISSUE TITLE
Crash when choose modify table

### ISSUE THREAD

**Issue by ivan-tretyak** [sqlitebrowser/sqlitebrowser#2457]
Details for the issue
--------------------

#### Step to reproduce
1.  Open the existing or create new database.
2. Create some table if you create new database.
3. Choose table for modify.

#### What did you expect to see?
Dialog for modify table.

#### What did you see instead?
Closing app.

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [x] Linux: ( _distro: Fedora 33 )
- [ ] macOS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [x] 3.12.0
- [ ] 3.11.x
- [ ] Other: ___

Backtrace: [backtrace.log](https://github.com/sqlitebrowser/sqlitebrowser/files/5440653/backtrace.log)

**Comment by xvitaly** [comment]
I can confirm this crash on Fedora 33. Can be reproduced by forwarding `-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS` build flags.

**Comment by ZaWertun** [comment]
![изображение](https://user-images.githubusercontent.com/1080298/97214847-bc06cd00-17d4-11eb-86c3-479ca77ee01d.png)
TLDR: app crashes with SIGABRT after clicking on highlighted menu item.

**Comment by nagayev** [comment]
@ivan-tretyak hello.
You can see my log in #2470.
Do you have similar log?

**Comment by MKleusberg** [comment]
Does this happen with any table or just with specific ones? Thanks for providing the stacktrace, @ivan-tretyak! This is going to be a huge help!

**Comment by MKleusberg** [comment]
This turned out to be a fairly obvious problem :smile: It should be solved now. Since you all use Linux you'll have to compile the latest version of the master branch by yourself. Do you feel like giving this a try? Many of our developers are Linux users too, so we can assist you when there are any problems while building the application.

**Comment by ivan-tretyak** [comment]
> Does this happen with any table or just with specific ones? Thanks for providing the stacktrace, @ivan-tretyak! This is going to be a huge help!

Yes, it's happening with any table

**Comment by justinclift** [comment]
@ivan-tretyak Is that with our latest development source code, which has a fix in it we're hoping works? :smile:

There are instructions for compiling our latest development source code, if that helps?

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/blob/master/BUILDING.md#centos--fedora-linux

Those instructions should (I think) still be fairly up to date. :smile:

**Comment by maclermo** [comment]
Looks like installing the latest build from source did the trick !

**Comment by justinclift** [comment]
Awesome! :smile:
